motionplan/collision.go

```diff
@@ -190,9 +190,9 @@ func (cg *collisionGraph) collisions() []Collision {
 	return collisions
 }
 
-// ignoreCollision finds the specified collision and marks it as something never to check for or report.
+// addCollisionSpecification marks the two objects specified as colliding.
 func (cg *collisionGraph) addCollisionSpecification(specification *Collision) {
-	cg.setDistance(specification.name1, specification.name2, math.NaN())
+	cg.setDistance(specification.name1, specification.name2, math.Inf(-1))
```

Comment:
We talked about changing this but I don't remember why?

Reply:
Comparisons with NaN are always false; but in this case we want it to conditionally register IsCollision as true, as we want to represent this as a collision in the reference.